README.MD

```diff
@@ -527,6 +527,54 @@ The according swagger definition can be merge-overwritten inside `tsoa.json`. He
 }
 ```
 
+### Tags
+
+If you have a project that needs a description and/or external docs for tags, you can configure the internal generators to use the correct tags definitions and external docs by providing a tags property to swagger property in tsoa.json.
+
+```json
+{
+  "swagger": {
+    "tags":  [
+      {
+        "name": "User",
+        "description": "Operations about users",
+        "externalDocs": {
+          "description": "Find out more about users",
+          "url": "http://swagger.io"
+        }
+      }
+    ],
+    ...
+  },
+  "routes": {
+    ...
+  }
+}
+```
+
+Tags are defined with the `@Tags('tag1', 'tag2', ...)` decorator in the controllers and/or in the methods like in the following examples.
```

Comment:
I think we should remove the part about the decorator being available also for controllers. It's true, but I don't yet understand what the outcome is. I guess there isn't one? Like.. it does nothing?